src/pd/client.rs

```diff
@@ -526,4 +528,37 @@ impl PdClient for RpcClient {
             .request(req, executor, LEADER_CHANGE_RETRY)
             .execute()
     }
+
+    fn get_timestamp(&self) -> PdFuture<u64> {
+        let timer = Instant::now();
+
+        let mut req = pdpb::TsoRequest::new();
+        req.set_header(self.header());
+        req.set_count(1);
+
+        let executor = move |client: &RwLock<Inner>, req: pdpb::TsoRequest| {
+            let option = CallOption::default().timeout(Duration::from_secs(REQUEST_TIMEOUT));
+            let (sender, receiver) = client.rl().client.tso_opt(option).unwrap();
+            let send = sender.send((req, WriteFlags::default().buffer_hint(false)));
+            Box::new(
+                send.and_then(|mut s| {
+                    let _ = s.close();
```

Comment:
We should not ignore the return value.

Reply:
Can we unwrap it? Or should we just print a error log for it?

Reply:
Can we use `cancel` here?